master/reference/calicoctl/resources/ippool.md

```diff
@@ -18,6 +18,7 @@ metadata:
 spec:
   ipip:
     enabled: false
+    mode: cross-subnet
```

Comment:
this is a bit confusing example. IPIP is disabled (`enabled: false`) AND mode is set to `cross-subnet`?